README.md

```diff
@@ -37,8 +37,8 @@ When deploying the multi cluster traffic controller using the make targets the f
 
 1. Build the controller image and load it into the control plane
     ```sh
-    export KUBECONFIG=./hack/kubeconfigs/mctc-control-plane.kubeconfig
-    make kind-load-controller
+   kubectl config use-context kind-mctc-control-plane 
+   make kind-load-controller
```

Comment:
Sticking with 1 KUBECONFIG, the users default, and just switching context is probably going to cause less issues. If you run `local-setup` in a terminal you have previously changed the KUBECONFIG in, it wouldn't work.

Reply:
It's annoying using context when you want to have  one terminal on a workload cluster and another on the control plane cluster, as changing context in one changes it in both.

Reply:
oh, I see, we're keeping the kubeconfig files, just not using them from the makefile

Reply:
true, if you want to configure 2 terminals one for each cluster, switching contexts isn't ideal. This is just docs changes, so not a huge issue either way but had it mess up a few times if you run local-setup after setting KUBECONFIG to one that gets pulled out of a kind cluster.